docs/en/stack/ml/df-analytics/dfa-classification.asciidoc

```diff
@@ -4,70 +4,100 @@
+set into training and testing data by specifying a `training_percent`. By
+default when you use the
+{ref}/put-dfanalytics.html[create {dfanalytics-jobs} API], 100% of the eligible
+documents in the data set are used for training. If you divide your data set,
+the job stratifies the data to ensure that both the training and testing data
+sets are representative of the full data set. That is to say, the same
+proportions of each class are fairly represented in both data sets.
+
+When you are collecting a data set to train your model, ensure that it
+captures information for all of the classes. If some classes are poorly
+represented in the training data set, the model might be unaware of them. In
+general, complex decision boundaries between classes are harder to learn and
+require more data points per class in the training data.
+
+////
+It means that you need to supply a labeled training data set that has a {depvar} 
+and some fields that are related to it. The {classification} algorithm learns 
+the relationships between these fields and the {depvar}. Once you’ve trained the 
+model on your training data set, you can reuse the knowledge that the model has 
+learned about the relationships between the data points to classify new data.
+
+The effects of imbalanced data are automatically mitigated before the 
+training. Nonetheless, it is a good idea to train your model with a data set 
+that is approximately balanced. That is to say, ideally your data set should 
+have a similar number of data points for each class.
+////
 
 [discrete]
 [[dfa-classification-algorithm]]
 ===== {classification-cap} algorithms
 
 //tag::classification-algorithms[]
-The ensemble algorithm that we use in the {stack} for {classification} is a type 
-of boosting called boosted tree regression model which combines multiple weak 
-models into a composite one. We use decision trees to learn to predict the 
-probability that a data point belongs to a certain class.
+{classanalysis-cap} uses an ensemble algorithm that is a type of boosting called 
+boosted tree regression model which combines multiple weak models into a 
+composite one. It uses decision trees to learn to predict the probability that a 
+data point belongs to a certain class. A sequence of decision trees are trained 
+and every decision tree learns from the mistakes of the previous one. Every tree 
+is an iteration of the last one, hence it improves the decision made by the 
+previous tree.
 //end::classification-algorithms[]
 
 
+[discrete]
+[[dfa-classification-performance]]
+==== {classification-cap} performance
+
+As a rule of thumb, a {classanalysis} with many classes takes more time to run 
+than a binary {classification} process when there are only two classes. The 
+relationship between the number of classes and the runtime is linear.
+
+The runtime also scales approximately linearly with the number of involved 
+documents below 200,000 data points. Therefore, if you double the number of 
```

Comment:
I think this paragraph might be a too much detail, because the exact conditions that we have when testing the runtime won't necessarily translate to the user's environment, so they might experience something different. I would leave in this sentence starting with `Therefore, if you double the number of  documents, ...`
I would remove this `Above 200,000 
data points, runtime increases faster.`, because it is a bit too specific.